npc/quests/quests_ein.txt

```diff
@@ -4741,7 +3263,6 @@ ein_in01,231,163,7	script	Sick Old Man#ein	4_M_DIEMAN,{
 		close;
 	}
 	else if (Shinokas_Quest == 2) {
```

Comment:
@dastgir This is outside of the Uwe Kleine NPC, I was only nice enough to correct typos, won't fix the code-style of the whole file in this PR. The whole file has a shitty code-style.